src/data/profiles.js

```diff
@@ -16,6 +17,13 @@ const getRandom = async (n = 8) => {
     return randUserData;
 }
 
+const getRandomNew = async (n = 8) => {
+    const randUserData = rawNew.sort(() => Math.random() - Math.random()).slice(0, n);
```

Comment:
btw kl manggil `rawNew.sort()` ky gini, bakal nge sort semua `rawNew` ya? sementara kita cuma butuh sebanyak `n`.

Gmn kalo bikin fungsi untuk ambil random index dari rawNew.length sebanyak n buah? Instead of looping sebanyak `rawNew.length` kali, cukup jadi `n` kali aja...

Reply:
Hmm, menarik, aku coba get random index, terus slice.
Footprint codenya lebih ringan, tapi dilihat randomness-nya lebih bagus dulu ga..

Reply:
Btw perlu make sure returned array nya unique aja sih... in case si `Math.floor(Math.random() * (rawNew.length+1))` menghasilkan `index` yang sama

Reply:
sudah diganti ke yang logic baru